libraries/datasource_okhttp/src/main/java/androidx/media3/datasource/okhttp/OkHttpDataSource.java

```diff
@@ -456,6 +464,10 @@ private void skipFully(long bytesToSkip, DataSpec dataSpec) throws HttpDataSourc
       }
       return;
     } catch (IOException e) {
+      if (e instanceof InterruptedIOException) {
+        closeConnectionQuietly();
```

Comment:
Same comment applies here as well. `skipFully` is only called from one place in this class, and that call site directly catches the exception thrown from L474 and immediately calls `closeConnectionQuietly()` already.

Reply:
I'll remove.